css-alpha-channel/index.html

```diff
@@ -0,0 +1,79 @@
+---
+feature_name: Alpha Channel Support in CSS Color Parsing
+chrome_version: 52
+feature_id: 5685348285808640
+---
+
+<h3>Background</h3>
+<p>Starting with Chrome 52, CSS rules now accept an alpha (transparency) channel for color values. Do this by specifying a fourth number in contexts that take an RGB value. Alpha channel support takes several forms.</p>
+<ul>
+  <li><code>#RRGGBBAA</code></li>
+  <li><code>#RGBA</code></li>
+  <li><code>rgba(RRR, GGG, BBB, A.A)</code></li>
```

Comment:
You should note that the `rgba()` syntax has wide legacy support: http://caniuse.com/#feat=css3-colors